gcs/src/dashboard.jsx

```diff
@@ -151,6 +209,9 @@ export default function Dashboard() {
     socket.on('incoming_msg', (msg) => {
       if (incomingMessageHandler[msg.mavpackettype] !== undefined) {
         incomingMessageHandler[msg.mavpackettype](msg)
+        if(msg.mavpackettype in possibleData === false){
```

Comment:
Can you add a comment explaining what this does pls

Reply:
okay